Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/RdbmsCentralRepo.java

```diff
@@ -1082,28 +1082,20 @@ public void addArtifactInstance(CorrelationAttributeInstance eamArtifact) throws
      */
     @Override
     public CentralRepoAccount getOrCreateAccount(CentralRepoAccountType crAccountType, String accountUniqueID) throws CentralRepoException {
-        // Get the account fom the accounts table
-        CentralRepoAccount account = getAccount(crAccountType, accountUniqueID);
+        String query = "INSERT INTO accounts (account_type_id, account_unique_identifier) "
+                + "VALUES ( " + crAccountType.getAccountTypeId() + ", '"
+                + accountUniqueID + "' ) ON CONFLICT DO NOTHING";
```

Comment:
The syntax for ON CONFLICT DO NOTHING is different for SQLite vs Postgres.  We need to be able to handle both. 

You would need to use the  method getPlatformSpecificInsertSQL(String sql)